tardis/model/base.py

```diff
@@ -72,7 +72,14 @@ def __init__(self, velocity, homologous_density, abundance, time_explosion,
         self.v_boundary_outer = v_boundary_outer
         self.homologous_density = homologous_density
         self._abundance = abundance
+        self.decayed = False
         self.time_explosion = time_explosion
+
+        self.raw_abundance = self._abundance
```

Comment:
@wkerzendorf If by `profile` , meaning time taken by decay function here
I made a dummy dataframe, then used decay on it.
It takes around 0.005 seconds for 10 elements and around 0.01 seconds for 100 elements